core/src/main/java/com/gentics/mesh/core/data/generic/AbstractMeshCoreVertex.java

```diff
@@ -37,21 +37,4 @@
 	protected <T> boolean shouldUpdate(T restValue, T graphValue) {
 		return restValue != null && !restValue.equals(graphValue);
 	}
-
-	@Override
-	public MeshElementEventModel createEvent(MeshEvent event) {
```

Comment:
`createEvent()` and `fillEventInfo` were duplicating the respective methods of `HibCoreElement`